tableauserverclient/models/view_item.py

```diff
@@ -186,4 +198,25 @@ def from_xml(cls, view_xml, ns, workbook_id="") -> "ViewItem":
             tags = TagItem.from_xml_element(tags_elem, ns)
             view_item.tags = tags
             view_item._initial_tags = copy.copy(tags)
+        if data_acceleration_config_elem is not None:
```

Comment:
should this check the API version before adding these elements? I assume earlier versions won't understand them.

Reply:
what's the best way to check the API version inside models object?
Do I need to pass in Server object and call check_at_least_version?

Reply:
hi @jacalata, pinging for advice how to do version check

Reply:
After evaluating the changes, I think adding versioning check in here is unnecessary since this function from_xml is only for parsing the response of the API call.
If using older api versions, calling get view will not return any data_acceleration_elem on the response & the code will check if the element is not None.